analyses/cell-type-consensus/references/consensus-validation-groups.tsv

```diff
@@ -1,53 +1,181 @@
+CL:1000488	cholangiocyte	CL:0000066	epithelial cell
+CL:0002305	epithelial cell of distal tubule	CL:0000066	epithelial cell
+CL:1000909	kidney loop of Henle epithelial cell	CL:0000066	epithelial cell
+CL:0000057	fibroblast	CL:0000057	fibroblast
+CL:0002410	pancreatic stellate cell	CL:0000057	fibroblast
+CL:0000125	glial cell	CL:0000125	glial cell
+CL:0000126	macroglial cell	CL:0000125	glial cell
+CL:0000095	neuron associated cell	CL:0000125	glial cell
+CL:0000681	radial glial cell	CL:0000125	glial cell
+CL:0000128	oligodendrocyte	CL:0000125	glial cell
+CL:0000636	Mueller cell	CL:0000125	glial cell
+CL:0000123	neuron associated cell (sensu Vertebrata)	CL:0000125	glial cell
+CL:0002453	oligodendrocyte precursor cell	CL:0000125	glial cell
+CL:0008001	hematopoietic precursor cell	CL:0008001	hematopoietic precursor cell
+CL:0000037	hematopoietic stem cell	CL:0008001	hematopoietic precursor cell
+CL:0002032	hematopoietic oligopotent progenitor cell	CL:0008001	hematopoietic precursor cell
+CL:0000051	common lymphoid progenitor	CL:0008001	hematopoietic precursor cell
+CL:0002031	hematopoietic lineage restricted progenitor cell	CL:0008001	hematopoietic precursor cell
+CL:0000038	erythroid progenitor cell	CL:0008001	hematopoietic precursor cell
+CL:0002274	histamine secreting cell	CL:0002274	histamine secreting cell
+CL:0001065	innate lymphoid cell	CL:0001065	innate lymphoid cell
+CL:0002681	kidney cortical cell	CL:1000497	kidney cell
+CL:1000497	kidney cell	CL:1000497	kidney cell
+CL:1000272	lung secretory cell	CL:1000272	lung secretory cell
+CL:0000235	macrophage	CL:0000235	macrophage
+CL:0000864	tissue-resident macrophage	CL:0000235	macrophage
+CL:1001603	lung macrophage	CL:0000235	macrophage
+CL:0000583	alveolar macrophage	CL:0000235	macrophage
+CL:0000091	Kupffer cell	CL:0000235	macrophage
+CL:0000863	inflammatory macrophage	CL:0000235	macrophage
+CL:0000861	elicited macrophage	CL:0000235	macrophage
+CL:0000129	microglial cell	CL:0000235	macrophage
+CL:0000650	mesangial cell	CL:0000650	mesangial cell
+CL:0000576	monocyte	CL:0000576	monocyte
+CL:0000192	smooth muscle cell	CL:0000187	muscle cell
+CL:0000187	muscle cell	CL:0000187	muscle cell
+CL:0000188	cell of skeletal muscle	CL:0000187	muscle cell
+CL:0000737	striated muscle cell	CL:0000187	muscle cell
+CL:0000746	cardiac muscle cell	CL:0000187	muscle cell
+CL:0000359	vascular associated smooth muscle cell	CL:0000187	muscle cell
+CL:0019019	tracheobronchial smooth muscle cell	CL:0000187	muscle cell
+CL:0000766	myeloid leukocyte	CL:0000763	myeloid
+CL:0000094	granulocyte	CL:0000763	myeloid
+CL:0000775	neutrophil	CL:0000763	myeloid
+CL:0000113	mononuclear phagocyte	CL:0000763	myeloid
+CL:0000764	erythroid lineage cell	CL:0000763	myeloid
+CL:0000556	megakaryocyte	CL:0000763	myeloid
+CL:0000771	eosinophil	CL:0000763	myeloid
+CL:0000233	platelet	CL:0000763	myeloid cell
+CL:0000232	erythrocyte	CL:0000763	myeloid cell
+CL:0000097	mast cell	CL:0000763	myeloid cell
```

Comment:
just checking - these ontologies are the same, are the annotations meant to be different (myeloid vs myeloid cell?)

Reply:
Thanks for spotting this! I think this is an error with joining the existing groups with the new groups where I must have been using myeloid instead of `myeloid cell`, which is the full cell ontology name. I'll update this to make sure the names are consistent with what's in Cell Ontology for all the validation groups.